_compdemos/intro-github.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Publishing code and data on GitHub
```

Comment:
I think we should focus this on code; data is another whole issue that needs to be covered separately